src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -13928,13 +13935,15 @@ void Player_Action_8084E9AC(Player* this, PlayState* play) {
 
 #if OOT_VERSION < PAL_1_0
         if (!LINK_IS_ADULT && LinkAnimation_OnFrame(&this->skelAnime, 5.0f)) {
+            // There is a jump sound when leaving the pedestal, but no landing sound when hitting the floor.
+            // This is fixed in PAL 1.0 and above.
             Player_PlayVoiceSfx(this, NA_SE_VO_LI_AUTO_JUMP);
         } else if (LINK_IS_ADULT) {
             func_8084E988(this);
         }
 #else
         if (!LINK_IS_ADULT) {
-            Player_ProcessAnimSfxList(this, D_808549F4);
+            Player_ProcessAnimSfxList(this, sJumpOffPedestalAnimSfxList);
         } else {
```

Comment:
Since you are inlining the data, why not put it here and avoid the extra version check?

```suggestion
        if (!LINK_IS_ADULT) {
            static AnimSfxEntry sJumpOffPedestalAnimSfxList[] = {
                { NA_SE_VO_LI_AUTO_JUMP, ANIMSFX_DATA(ANIMSFX_TYPE_VOICE, 5) },
                { 0, -ANIMSFX_DATA(ANIMSFX_TYPE_LANDING, 15) },
            };

            Player_ProcessAnimSfxList(this, sJumpOffPedestalAnimSfxList);
        } else {
```

Reply:
just didnt think of it :D